src/dao/api.dao.ts

```diff
@@ -248,7 +255,12 @@ export class NodeDao {
 
   getFeedEntries = async (offset: number, limit: number, search: string) => {
 
-    const baseWhereClause = `WHERE NOT (n1."entType" = 'CLAIM' AND e.label = 'source')`;
+    if (typeof search === 'string') {
+      // for some reason they are getting double-encoded 
+      search = decodeURIComponent(search);
+    }
+
+    const baseWhereClause = `WHERE NOT (n1."entType" = 'CLAIM' AND e.label = 'source') AND NOT (c."effectiveDate" is null) and NOT (n1.name is null or n1.name = '') `;
```

Comment:
need this because sometimes effectiveDate is null

Reply:
used it too in this PR => https://github.com/Whats-Cookin/trust_claim_backend/pull/121